ITAQA/crawler/emilia_romagna.py

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Emilia-Romagna data downloader and parser
+"""
+
+from datetime import datetime, timedelta
+from functools import reduce
+import urllib
+import pandas as pd
+
+from ITAQA.core.Pollutants import Pollutant
+
+
+def get_csv(pollutant=Pollutant.UNSET, days=10):
+    """
+    Return a csv containing the latest N days of measurament of Emilia-Romagna
+    air quality (pollutant of choice)
+
+    Data origin: ARPA Emilia-Romagna
+    Website: https://apps.arpae.it/qualita-aria/bollettino-qa
+    """
+
```

Comment:
Minor thing, only mentioned now and not in the future as not really relevant: after "TODO:" prefer capital first letter

Reply:
Wilco!